src/App.js

```diff
@@ -20,22 +22,33 @@ function App() {
 
   // TODO -- add event for button click to handle calling fetchBusinesses with zip / search
 
+  const searchRestaurants = async () => {
+    const resp = await fetch(`/.netlify/functions/fetch-yelp?zip=${zip}`);
+    const data = await resp.json();
+    setBusinesses(data);
+  };
+
+  const filterBusinesses = () => {
+    const filteredBusinesses = businesses.filter(business => business.name.toLowerCase().includes(search));
+    return filteredBusinesses;
+  };
```

Comment:
Nice use of a filter here, but I believe the search state was meant to be included in the fetch request to the Yelp API. Consider adding a new input for filtering and an associated `filter` state for this functionality.